pkg/rule/rule.go

```diff
@@ -33,11 +37,115 @@ func (g Group) OriginalFile() string {
 	return g.originalFile
 }
 
+// GetRules returns a list of rules, which is used in UI rules page.
+func (g *Group) GetRules() []rules.Rule {
+	var rs []rules.Rule
+	for _, rule := range g.Rules() {
+		switch r := rule.(type) {
+		case *rules.AlertingRule:
+			rs = append(rs, &AlertingRule{r, g.PartialResponseStrategy})
+		case *rules.RecordingRule:
+			rs = append(rs, &recordingRule{r, g.PartialResponseStrategy})
```

Comment:
having AlertingRule exported and recordingRule unexported looks strange, I think it would be good to have both exported or unexported for consistency :)

Exporting it allows other users to use it as package, unexporting it hides away from that. Not sure which one is the best in this case tbh

Reply:
Since AlertingRule must be public, I will make it recordingRule public as well and add a comment